client/hooks/useURLUpdate.ts

```diff
@@ -33,7 +33,7 @@ export function useURLUpdate() {
 
     url.searchParams.set("page", "1");
 
-    self.history.pushState({}, "", url.toString());
+    // self.history.pushState({}, "", url.toString());
```

Comment:
not sure the point of history.push when below this it forces a full page refresh. What do we want? to keep things client side or do full page refreshes?

Reply:
created an issue for this. thanks for spotting

https://github.com/stampchain-io/BTCStampsExplorer/issues/524